src/fsharp/FSharp.Core/seq.fsi

```diff
@@ -292,35 +292,50 @@ module Seq =
     ///
     /// <example id="compare-with-1">
     /// <code lang="fsharp">
-    /// ([1; 2], [1; 2]) ||> Seq.compareWith (fun a b -> a.CompareTo(b))
+    /// let closerToNextDozen a b =
+    ///   (a % 12).CompareTo(b % 12)
+    /// ([1; 10], [1; 10])
```

Comment:
Please put `||>` all on one line all through these.  Also add a blank line either side of the actual example

Reply:
Done